### PR TITLE
AC: Added age_recognition adapter

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/adapters/attributes_recognition.py [hunk 1]
```diff
@@ -133,8 +133,8 @@ class AgeGenderAdapter(Adapter):
     def parameters(cls):
         parameters = super().parameters()
         parameters.update({
-            'age_out': StringField(description="Output layer name for age recognition."),
-            'gender_out': StringField(description="Output layer name for gender recognition.")
+            'age_out': StringField(description="Output layer name for age recognition.", optional=True),
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/adapters/attributes_recognition.py:136]: why these parameters become optional?

File: tools/accuracy_checker/accuracy_checker/adapters/attributes_recognition.py [hunk 2]
```diff
@@ -179,6 +179,26 @@ def process(self, raw, identifiers=None, frame_meta=None):
         return result
 
 
+class AgeRecognitionAdapter(AgeGenderAdapter):
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/adapters/attributes_recognition.py:182]: not sure that  inheritance logic is correct (should be reversed as the rule).

File: tools/accuracy_checker/accuracy_checker/adapters/attributes_recognition.py (hunk 1 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/adapters/attributes_recognition.py]: returned the previous state

File: tools/accuracy_checker/accuracy_checker/adapters/attributes_recognition.py (hunk 2 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/adapters/attributes_recognition.py]: Updated

**Review by eaidova — APPROVED** [review]

**Pull request merged**